### DISCUSSION TITLE
Contribution question

### DISCUSSION THREAD

**Discussion by trontronicent** [justcoding121/Titanium-Web-Proxy#880] [General]
Hey there
I use this awesome code library so intensively, that I thought, the least I should do, is to contribute now something. 
Regarding this:

Is there a reason the Api Doc is not also integrated into the assembly? I had to copy some parts of the Api site into the code, to not have to switch out of the IDE all the time, and I was now wondering if I could contribute by bringing the whole Api doc in form of XML doc comments into. But before I spend that time, i wanted to make sure, its not left out on purpose.

Keep up the great work
RK

**Comment by justcoding121** [comment]
Not left out on purpose ? contributions for XML docs are very welcome.

If you want to bring API docs into the assembly as `///` comments, please do. A few notes:
- Keep comments accurate to current behavior (the public site can lag).
- Prefer short, useful summaries over dumping the whole website into every member.
- PRs are appreciated; smaller focused ones are easier to review.

Thanks for offering ? looking forward to it. Closing this as answered.
